cmd/kubeless/autoscale/autoscaleCreate.go

```diff
@@ -24,7 +24,12 @@ var autoscaleCreateCmd = &cobra.Command{
 			ns = utils.GetDefaultNamespace()
 		}
 
-		function, err := utils.GetFunction(funcName, ns)
+		kubelessClient, err := utils.GetKubelessClientOutCluster()
+		if err != nil {
+			logrus.Fatalf("Can not out-of-cluster client: %v", err)
```

Comment:
Suggestion: `logrus.Fatalf("Unable to instantiate the kubeless client: %v", err)`